### PR TITLE
Add jaimemendozadev-los-angeles.md 

### LINKED ISSUE #39: jaimemendozadev Workflow for Adding Los Angeles to the Repo
## Los Angeles, CA
- [ ] Create an Issue
- [ ] Create a Branch
- [ ] Make Changes
- [ ] Open a PR
- [ ] Collaborate/Have a Discussion
- [ ] Get Approval to 👍 
- [ ] Merge

### PR DESCRIPTION
**Pull request by jaimemendozadev**
@githubteacher 
- Add @jaimemendozadev-los-angeles.md 
- Populate file with info (by all means look at my restaurant choices) 😉 

closes #39

### CODE REVIEW

File: jaimemendozadev-los-angeles.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+##Los Angeles, CA
+
+### Restaurants
+
++ [Tsujita](https://www.yelp.com/biz/tsujita-la-artisan-noodle-los-angeles-2)
++ [In-N-Out Burger](http://www.in-n-out.com/)
```
Comment (githubteacher) [inline jaimemendozadev-los-angeles.md:6]: Do you prefer In N Out or 5 Guys?

**Comment by githubteacher** [comment]
IT IS TOO HOT

File: jaimemendozadev-los-angeles.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
++ [Langer's Delicatessen](http://www.langersdeli.com/)
++ [Cassell's Hamburgers](http://www.cassellshamburgers.com/)
+
+### Places to Go
++ Dodger Stadium
++ [The Getty](https://www.getty.edu/)
++ [Disneyland](https://disneyland.disney.go.com/destinations/disneyland/)
+
```
Comment (githubteacher) [inline jaimemendozadev-los-angeles.md:14]: Where would you recommend that I go on a hot day?

Comment (makdevio) [inline jaimemendozadev-los-angeles.md:14]: like to check out some day

File: jaimemendozadev-los-angeles.md (hunk 1 above)
Comment (jaimemendozadev) [reply jaimemendozadev-los-angeles.md:6]: LOL, I'm personally an In N Out kind of guy.

Five Words: Four-by-four animal style. 👍 😄

File: jaimemendozadev-los-angeles.md (hunk 2 above)
Comment (jaimemendozadev) [reply jaimemendozadev-los-angeles.md:14]: LOL, only go to Disneyland once and never go back. They're raising the prices for the annual pass this year I think.

Comment (jaimemendozadev) [reply jaimemendozadev-los-angeles.md:14]: LOL, for a minute I thought you meant where you can go on a hot date. 

Honestly, on a hot day, I'd stay on indoors. If you're on the West Side (by which I mean anything past West of La Brea Ave.), you should be okay being able to walk on the street without sweating too much.

The closer you get to the ocean, the less hot I think you'll be.

**Review by rice-ball-rolling — APPROVED** [review]
nice

**Pull request merged**